python_modules/libraries/dagster-databricks/README.md

```diff
@@ -2,3 +2,146 @@
 
 The docs for `dagster-databricks` can be found
 [here](https://docs.dagster.io/_apidocs/libraries/dagster-databricks).
+
+## EXT Example
+
+This package includes a prototype API for launching databricks jobs with
+Dagster's EXT protocol. There are two ways to use the API:
+
+### (1) `ExtDatabricks` resource
+
+The `ExtDatabricks` resource provides a high-level API for launching
+databricks jobs using Dagster's EXT protocol.
+
+It takes a single `databricks.sdk.service.jobs.SubmitTask` specification. After
+setting up EXT communications channels (which by default use DBFS), it injects
+the information needed to connect to these channels from Databricks into the
+task specification. It then launches a Databricks job by passing the
+specification to `WorkspaceClient.jobs.submit`. It polls the job state and
+exits gracefully on success or failure:
```

Comment:
i think we should lowercase ext. It's not principled, but it looks cooler

Reply:
I might be wrong/misguide on this

Reply:
let's discuss in standup

Reply:
We didn't end up discussing but I changed it. Personally am ambivalent.